resources/views/models/product-variants/form.blade.php

```diff
@@ -0,0 +1,163 @@
+<x-fab::layouts.page
+    :title="$model?->parentProduct?->title ?: 'Untitled'"
+    :breadcrumbs="[
+            ['title' => 'Home', 'url' => route('lego.dashboard')],
+            ['title' => 'WG Product Match', 'url' => route('lego.go-load-up.product-match.index')],
+            ['title' => $model?->parentProduct?->title ?: 'Untitled'],
+        ]"
+    x-data=""
+    x-on:keydown.meta.s.window.prevent="$wire.call('save')"  For Mac
+    x-on:keydown.ctrl.s.window.prevent="$wire.call('save')"  For PC
+>
+    <x-slot name="actions">
+        @include('lego::models._includes.forms.page-actions')
+    </x-slot>
+    <x-lego::feedback.errors class="sh-mb-4" />
+
+    <x-fab::layouts.main-with-aside>
+        <x-fab::layouts.panel>
+            <x-fab::forms.select
+                wire:model="model.parent_product_variant_id"
+                wire:change="setErrorLabel"
+                label="Parent Product Variant/Service"
+                help="Choose a product variant for which you want to set rules. This will be used in the Cart page; when users attempt to proceed to checkout, the required products for each GoLoadUp services in the cart would be checked if they are available."
+            >
+                <option value="">-- Select GoLoadUp Product Variant --</option>
+                @foreach($this->getVariantsOfDefaultProduct() as $variant)
+                    <option value="{{ $variant->id }}">{{ $variant->title }}</option>
+                @endforeach
+            </x-fab::forms.select>
+            <x-fab::forms.input
+                label="Error Message"
+                wire:model="model.error_message"
+                help="Displays when a product is missing from one of the lists below."
+
+            />
+        </x-fab::layouts.panel>
+
+        <div class="grid grid-cols-2 gap-4">
+            <x-fab::layouts.panel
+                title="Product Array 1"
+                description="Link products to this parent product"
+                class="sh-mt-4"
+                allow-overflow
+                x-on:fab-added="$wire.call('selectProduct', $event.detail[1].key, 1)"
+                x-on:fab-removed="$wire.call('unselectProduct', $event.detail[1].key, 1)"
+            >
+                @if($model->exists)
+                    <x-fab::forms.combobox
+                        :componentKey="444444"
```

Comment:
@OniiCoder Awesome job on this PR. Care to explain why `componentKey="4444"`. Thanks 😄

Reply:
lol @iamkarsoft I think it unique key for that component and he decided to use 4444...maybe its his lucky number😂
I can also see componentKey='33' in the next component

Reply:
😂😂😂😂😂😂 Well, it was a random unique number that came to my mind.